### PR TITLE
add ambrosus

### PR DESCRIPTION
**Pull request by VoroninDima**

### CODE REVIEW

File: _data/chains/eip155-1109.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "shortName": "amb",
+  "chainId": 30741,
+  "networkId": 30741,
+  "icon": "ambrosus",
+  "explorers": [
+    {
+      "name": "Ambrosus",
+      "url": "https://explorer.ambrosus.io/",
+      "icon": "ambrosus",
+      "standard": ""
```
Comment (ligi) [inline _data/chains/eip155-1109.json:20]:
```suggestion
      "url": "https://explorer.ambrosus.io",
      "icon": "ambrosus",
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]
the  chainID in the json does not match the filename - also the comments I made as suggestion to fix the explorer section

**Comment by VoroninDima** [comment]
> the chainID in the json does not match the filename - also the comments I made as suggestion to fix the explorer section

iIaccepted your suggestion and fixed filename

**Comment by ligi** [comment]
Thanks, but you need to fix the icon PR first so I can merge this one - otherwise the CI won't let me. You can also make the Icon part of this PR.

**Comment by VoroninDima** [comment]
> Thanks, but you need to fix the icon PR first so I can merge this one - otherwise the CI won't let me. You can also make the Icon part of this PR.

ready

**Pull request closed without merging**